packages/server/src/unstable-core-do-not-import/rpc/envelopes.ts

```diff
@@ -99,6 +99,7 @@ export type TRPCClientOutgoingMessage =
 
 export interface TRPCResultMessage<TData>
   extends JSONRPC2.ResultResponse<
+    | { type: 'connecting'; data?: never }
```

Comment:
I don't think this should be in the rpc spec since it doesn't affect actual communication between server/client